package.json

```diff
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/y-scope/yscope-log-viewer#readme",
   "dependencies": {
-    "@monaco-editor/react": "^4.4.6",
+    "monaco-editor-webpack-plugin": "^7.1.0",
```

Comment:
Sort the properties. e.g., In WebStorm, you can 
![image](https://github.com/y-scope/yscope-log-viewer/assets/43196707/e8403083-a083-4ac8-bb7e-ef12e1c17f9a)